Todo.py

```diff
@@ -0,0 +1,203 @@
+import discord
+from settings import *
+from checks import *
+
+TOKEN = "NTY4MDkyNTY5NTE5NzgzOTM2.XMiBDw.zCL0KhV_gOtxfyACctgYCLcLlEY"
```

Comment:
@adrienluitot you should delete this token

Reply:
This line should be deleted before I can merge...

Reply:
This line should be deleted before I can merge...

Reply:
Already done ;)

Reply:
Why am I not seeing it then ? You haven't you committed ?

Reply:
it's that